### PR TITLE
Added functionality for adding citations to a filter + generating a bibtex file from a pipeline

### PR DESCRIPTION
**Pull request by wlenthe**

### CODE REVIEW

File: Source/Applications/DREAM3D/DREAM3DApplication.h [hunk 1]
```diff
@@ -128,6 +128,7 @@ class DREAM3DApplication : public QApplication
     void on_actionShowPrebuiltInFileSystem_triggered();
     void on_actionLocateFile_triggered();
     void on_actionClearPipeline_triggered();
+    void on_actionActionGeneratePipelineBibliography_triggered();
```
Comment (imikejackson) [inline Source/Applications/DREAM3D/DREAM3DApplication.h]: Why is the word "Action" used twice?

**Comment by imikejackson** [comment]
This probably should have been written as a Filter that one can tack onto the end of a pipeline. Not sure I agree with putting a Menu Item into the main menu item. I'll let others chime in with their thoughts.

**Comment by wlenthe** [comment]
I don't really have a preference for where the functionality lives as long as it exists somewhere (I've added an example use to EM/MPM if that helps decide on menu vs. filter)

File: Source/Applications/DREAM3D/DREAM3DApplication.h (hunk 1 above)
Comment (wlenthe) [inline Source/Applications/DREAM3D/DREAM3DApplication.h]: this was a copy paste error (corrected by commit 090380b)

**Comment by wlenthe** [comment]
moved to new branch in pull request #438

**Pull request closed without merging**